src/components/pages/add-revision.tsx

```diff
@@ -16,6 +17,23 @@ export function AddRevision({
 }: EditorPageData) {
   const { strings } = useInstanceData()
 
+  const [cookieReady, setCookieReady] = useState(false)
+
+  useEffect(() => {
+    if (window.location.hostname === 'localhost') {
+      setCookieReady(true)
+    } else {
+      fetch('/auth/password/change')
+        .then((res) => res.text())
+        .then(() => {
+          setCookieReady(true)
+        })
+        .catch(() => {})
```

Comment:
Possible optimization: check if the cookie already exists.

Reply:
sure it can not be outdated or smth?

Reply:
will add optimization later via staging.